src/components/TheTopNavbar.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-toolbar data-cy="nav-drawer" color="primary">
+    <v-toolbar-title>
+      <div class="d-flex flex-md-row flex-row-reverse align-center justify-space-between" style="cursor: pointer">
+        <TheUserMenu />
+        <img
+          id="logo"
+          alt="Cuttle logo"
+          src="/img/cuttle_logo_text_white.svg"
+          width="60"
+          height="60"
+          class="ma-md-auto"
+        />
+      </div>
+    </v-toolbar-title>
+    <v-toolbar-items v-if="!isSmallDevice" class="hidden-xs-only">
+      <v-tabs class="pa-2">
+        <v-tab
+          variant="text"
+          v-for="({ text, icon, page }, i) in pageLinks"
+          :data-cy="text"
+          :key="i"
+          :title="text"
+          :to="page"
+        >
+          <v-icon :icon="`mdi-${icon}`" />
```

Comment:
This icon also needs a title or text to denote what it is. Alternatively you can use aria hidden to have screen readers ignore it so that the descriptive text doesn't matter.

Reply:
I think that we want to set a role, aria-label, and aria-hidden="false". 
```suggestion
          <v-icon
            :icon="`mdi-${icon}`"
            :aria-label="text"
            aria-hidden="false"
            role="img" 
          />
```

The label is for screen readers to know what the icon's about, aria-hidden="false" ensures it appears for screen readers, and role="img" tells screen readers that this icon represents an image

Reply:
To play devil's advocate the icon doesn't add anything for a screen reader that the text doesn't so just hiding it is less work imo.

Reply:
We can normalize this in a future app-wide accessibility update